modules/aws/jumpbox/jumpbox.userdata

```diff
@@ -21,19 +22,41 @@ users:
 
 write_files:
   - content: |
-      #!/bin/sh
-      curl -Lo "/usr/local/bin/tctl" "https://binaries.dl.tetrate.io/public/raw/versions/linux-amd64-${tsb_version}/tctl"
-      chmod +x "/usr/local/bin/tctl"
-      cp /usr/local/bin/tctl /usr/share/nginx/html/
+      #!/bin/bash
       curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
       install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl
       ${docker_login}
-      tctl install image-sync \
-        --username "${tsb_image_sync_username}" \
-        --apikey "${tsb_image_sync_apikey}" \
-        --registry "${registry}" \
-        --accept-eula \
-        --parallel
+      export PROJECT_CR="${registry}"
+      # to initialize GCR and use skopeo for TSB images sync below
+      docker pull busybox
+      docker tag busybox $PROJECT_CR/busybox
+      docker push $PROJECT_CR/busybox
+      curl https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3 | bash
+      helm repo add tetrate-tsb-helm 'https://charts.dl.tetrate.io/public/helm/charts/'
+      helm repo update
```

Comment:
Can we move the helm bits inside the development conditional? We shouldn't need it for normal tctl-based installs.

Reply:
ack, done